phoenix-core/src/main/java/org/apache/phoenix/util/ScanUtil.java

```diff
@@ -956,8 +1026,130 @@ public static int getClientVersion(Scan scan) {
         }
         return clientVersion;
     }
-    
+
     public static void setClientVersion(Scan scan, int version) {
         scan.setAttribute(BaseScannerRegionObserver.CLIENT_VERSION, Bytes.toBytes(version));
     }
+
```

Comment:
Most of the diffs in ScanUtil are formatting changes. Can we revert them?

Reply:
I think it has to do with IDE auto formating